cmd/ore/equinixmetal/create-device.go

```diff
@@ -45,7 +45,7 @@ func init() {
 	cmdCreateDevice.Flags().StringVar(&options.InstallerImageBaseURL, "installer-image-base-url", "", "installer image base URL, non-https (default board-dependent, e.g. \"http://stable.release.flatcar-linux.net/amd64-usr/current\")")
 	cmdCreateDevice.Flags().StringVar(&options.InstallerImageKernelURL, "installer-image-kernel-url", "", "EquinixMetal installer image kernel URL, (default installer-image-base-url/flatcar_production_pxe.vmlinuz)")
 	cmdCreateDevice.Flags().StringVar(&options.InstallerImageCpioURL, "installer-image-cpio-url", "", "EquinixMetal installer image cpio URL, (default installer-image-base-url/flatcar_production_pxe_image.cpio.gz)")
-	cmdCreateDevice.Flags().StringVar(&options.ImageURL, "image-url", "", "image base URL (default board-dependent, e.g. \"https://alpha.release.flatcar-linux.net/amd64-usr/current/flatcar_production_packet_image.bin.bz2\")")
+	cmdCreateDevice.Flags().StringVar(&options.ImageURL, "image-url", "", "image base URL (default board-dependent, e.g. \"https://alpha.release-linux.flatcar.net/amd64-usr/current/flatcar_production_packet_image.bin.bz2\")")
```

Comment:
```suggestion
	cmdCreateDevice.Flags().StringVar(&options.ImageURL, "image-url", "", "image base URL (default board-dependent, e.g. \"https://alpha.release-linux.flatcar-linux.net/amd64-usr/current/flatcar_production_packet_image.bin.bz2\")")
```
same

Reply:
Ah I messed up.
It should be:
```
https://alpha.release.flatcar-linux.net
```

I appended `-linux` to `release` instead of `flatcar`. :expressionless:

Reply:
Oh, right, updated the suggestion